flintrock/templates/spark/conf/spark-defaults.conf

```diff
@@ -0,0 +1 @@
+spark.jars.packages    org.apache.hadoop:hadoop-aws:{hadoop_version}
```

Comment:
I don't think `--jars` and `--packages` do exactly the same thing. I believe `--packages` will load the transitive dependencies automatically, whereas `--jars` will not.

If you try Spark 2.0 or 2.1 as follows:

```
spark-shell --packages "org.apache.hadoop:hadoop-aws:2.7.3"
```

Do you still have problems accessing S3?

Earlier you said you would get this error `java.lang.ClassNotFoundException: com.amazonaws.auth.AWSCredentialsProvider`, but since you mentioned `--jars` later on I want to make sure you were actually using `--packages` when you got that first error.

If you still get this error when testing locally, my next question is, do you have any problems accessing S3 using Flintrock as described in the README in this PR?